types/office-js-preview/index.d.ts

```diff
@@ -18588,6 +18588,55 @@ declare namespace Excel {
          */
         getRangeOrNullObject(ctx: Excel.RequestContext): Excel.Range;
     }
+    /**
+     *
+     * Provides information about the worksheet format change event.
```

Comment:
Seems like this should be something like:
```suggestion
     * Provides information about the worksheet row hidden change event.
```